openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -34,6 +34,19 @@ def generate_zulu_datetime():
     return fake.date_time().isoformat() + 'Z'
 
 
+def generate_price_ranges():
+    return [{
+        'currency': 'USD',
+        'max': 1000,
+        'min': 100,
+        'total': 500
+    }]
+
+
+def generate_seat_sku():
+    return 'sku_{}'.format(str(generate_zulu_datetime())[:10])
```

Comment:
I'll implement this the way Vedran suggested.
I did it this way because I went to Django Oscar source code to see how they do it. 
That was a moment of temporal insanity.